index.js

```diff
@@ -1,6 +1,93 @@
 /* jshint node: true */
 'use strict';
 
+var Funnel = require('broccoli-funnel');
+var walkSync = require('walk-sync');
+var path = require('path');
+var fs = require('fs');
+
+function normalize(name) {
+  if (typeof name === 'string') {
+    name = path.basename(name, '.js');
+
+    return name.replace(/([a-z](?=[A-Z]))/g, '$1-').toLowerCase();
+  }
+}
+
+function unique(arr) {
+  var out = [];
+
+  for (var i=0;i<arr.length;i++) {
+    var obj = arr[i];
+
+    if (obj && out.indexOf(obj) === -1) {
+      out.push(obj);
+    }
+  }
+
+  return out;
+}
+
+function getEach(arr, propName) {
+  var out = [];
+
+  for (var i=0;i<arr.length;i++) {
+    var obj = arr[i];
+
+    if (obj && obj[propName]) {
+      out.push(obj[propName]);
+    }
+  }
+
+  return out;
+}
+
 module.exports = {
-  name: 'ember-metrics'
+  name: 'ember-metrics',
+
+  included: function(app) {
+    this._super.included.apply(this, arguments);
+
+    var config = this.app.project.config(app.env) || {};
+    var addonConfig = config[this.name] || {};
+    var applicationAdapters = path.join(app.options.project.root, app.trees.app, 'metrics-adapters');
```

Comment:
We need to walk the application metrics-adapters directory and assume anything that is there should be considered a whitelisted adapter so that treeForApp does not filter it out.

This will fail in the case where a user is extending an ember-metrics adapter w/o including it.  Unsure of how to properly solve this that doesn't involve inspecting the import statements.